src/VisualStudio/VisualBasic/Impl/BasicVSResources.resx

```diff
@@ -300,4 +300,7 @@
   <data name="Show_items_from_unimported_namespaces" xml:space="preserve">
     <value>Show items from unimported namespaces (experimental)</value>
   </data>
+  <data name="VisualBasic_language_server_client" xml:space="preserve">
+    <value>VisualBasic Language Server Client</value>
```

Comment:
```suggestion
    <value>Visual Basic Language Server Client</value>
```

Reply:
sure. I can make that change. I see VB CodeSytle thing as well. should I change that to Visual Basic as well then?